package.json

```diff
@@ -18,10 +18,32 @@
     "build": "react-scripts build",
     "deploy": "aws s3 sync build/ s3://biztech-web",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint .",
+    "format": "prettier --write \"**/*.+(js|jsx|json|yml|yaml|css|md|vue)\""
```

Comment:
are these scripts ever run automatically? Or do we have to run them ourselves

Reply:
for manual running:
- `npm run lint` uses eslint to check code-quality of all files
- `npm run format` uses prettier to format all files that end with the listed suffixes

will note in mention